numbers/incompleteBeta.go

```diff
@@ -0,0 +1,51 @@
+package numbers
+
+import (
+	"log"
+	"math"
+	"math/rand"
```

Comment:
Can you run a go test in the numbers package for me? I am getting an error message that says `./incompleteBeta.go:6:2: imported and not used: "math/rand"` when i checked out your branch. I think we should just remove `"math/rand"` but I just want to make sure.

Reply:
Yes, this is removed in the newest commit and numbers now passes tests.